.github/workflows/ci.yml

```diff
@@ -143,7 +143,7 @@ jobs:
     - name: Install llvmpipe and lavapipe for offscreen canvas
       run: |
         sudo apt-get update -y -qq
-        sudo apt-get install --no-install-recommends -y libegl1-mesa-dev libgl1-mesa-dri libxcb-xfixes0-dev mesa-vulkan-drivers
```

Comment:
We were using a mix of commands with/without this flag. I removed it in all.

Reply:
Depending on what you install, it can save a lot of time to include the flag. But in our case I don't think any recommended extras are being installed, so it's fine.

Reply:
Thanks for checking. If it would, we should probably update this line everywhere to be consistent :)